tests/end_to_end/localstack/Dockerfile

```diff
@@ -6,13 +6,13 @@ WORKDIR /dependencies
 ENV PIPENV_VENV_IN_PROJECT=1
 
 COPY --from=root_dir .env.test /.env
-COPY --from=root_dir Pipfile /
-COPY --from=root_dir Pipfile.lock /
+COPY --from=root_dir pyproject.toml /
```

Comment:
Can we just delete the entire dir `tests/end_to_end/localstack/` as we don't use it (I added this accidentally)